### DISCUSSION TITLE
Custom element  on Navigation bar

### DISCUSSION THREAD

**Discussion by SO-Matveev** [maplibre/maplibre-gl-js#4423] [Q&A] [answered]
How i can to add custom element on Navigation bar (Navigation Control), for example i need add element which show zoom level. Unfortunately i do not find in documentation how do it. Thanks for answer!

**Comment by HarelM** [comment, marked as answer]
I would recommend creating a control and use addControl instead of trying to hack the current control.

**Reply by SO-Matveev** [reply]
Thank you for responding, I will try =)
